scripts/fix_wheel.py

```diff
@@ -9,45 +9,49 @@
 
     # Get the file from the dist dir
     dist_files = glob("dist/*.whl")
-    if len(dist_files) != 1:
-        raise RuntimeError("Should have single wheel file in dist")
-    existing_wheel_file = dist_files[0]
+    if not dist_files:
+        raise RuntimeError("Should have at least one wheel file in dist")
+    for existing_wheel_file in dist_files:
+        # Rename the wheel file and confirm it is changed. We need to make it
+        # py37 minimum interpreter and abi3 compat.
+        wheel_file_pieces = existing_wheel_file.split("-")
+        if len(wheel_file_pieces) < 4:
```

Comment:
nit:
Can this be > 4?
If it can't you can destructure into `part1, part2, _py, _abi` and replace the join with `f"{part1}-{part2}-cp37-abi3"` (replace part1 and part2 with meaningful names)